README.md

```diff
@@ -19,10 +19,49 @@ A Julia package for probability distributions and associated functions. Particul
 **Note:** The functionalities related to conjugate priors have been moved to the [ConjugatePriors package](https://github.com/JuliaStats/ConjugatePriors.jl).
 
 
-#### Resources
+## Resources
 
 * **Documentation**: <https://JuliaStats.github.io/Distributions.jl/stable/>
 
+## Contributing
+
+### Reporting issues & discussing
+
+1. If you need explanation on how to do X, Y using Distributions,
+feel free to ask on the Julia [Discourse](https://discourse.julialang.org/c/domain/stats)
+or [Slack](https://julialang.slack.com), get an invitation
+[here](https://slackinvite.julialang.org/).
+
+2. If you have a bug linked with Distributions, check that it has
+not been reported yet on the issues of the repository.
+If not, you can file a new issue, add your version of the package
+which you can get with this command in the Julia REPL:
+```julia
+julia> import Pkg
+julia> Pkg.status("Distributions")
```

Comment:
I'd just recommend the Pkg REPL mode: `]status Distributions`

Reply:
fixed in last commit thanks